gems/pending/appliance_console/database_replication.rb

```diff
@@ -0,0 +1,115 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'English'
+
+module ApplianceConsole
+  class DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REPMGR_CONFIG = '/etc/repmgr.conf'.freeze
+
+    attr_accessor :cluster_name, :node_number,
+                  :database_name, :database_user, :database_password,
+                  :primary_host, :standby_host
+
+    def ask_for_cluster_name
+      self.cluster_name = just_ask("cluster name", cluster_name)
+    end
+
+    def ask_for_node_number
+      self.node_number = ask_for_integer("A number uniquely identifying this node in the replication clusternode")
```

Comment:
@carbonin,

Thanks for the review feedback. I'm not done making the updates we discussed yesterday. I just wanted to push what I had to github when I left for the day yesterday. I'll add a comment when I am ready for you to review my updates.